### PR TITLE
Initialize save location in the footer status bar #311

### LINKED ISSUE #311: Initialize save location in the footer status bar
The initialization got removed in a #307

### PR DESCRIPTION
**Pull request by yl-coder**
Fixes #311

### CODE REVIEW

File: src/main/java/address/controller/MainController.java [hunk 1]
```diff
@@ -185,7 +183,7 @@ private void showFooterStatusBar() {
             GridPane gridPane = loader.load();
             gridPane.getStyleClass().add("grid-pane");
             StatusBarFooterController controller = loader.getController();
-            controller.initStatusBar(config);
+            controller.initStatusBar(config, modelManager.getPrefs());
```
Comment (HansNewbie) [inline src/main/java/address/controller/MainController.java]: Should we name the method as `setConnections`? To be consistent with other controllers. If there is other initialization needed, maybe have `start` after `setConnections` or put in constructor.

Comment (yl-coder) [inline src/main/java/address/controller/MainController.java]: changed to init()

File: src/main/java/address/controller/StatusBarFooterController.java [hunk 2]
```diff
@@ -57,27 +57,28 @@ private void syncSchedulerOnTimeOut() {
...
-    public void initStatusBar(Config config) {
-        updateWithConfigValues(config);
+    public void init(long updateInterval, String saveLocation) {
+        initSyncTimer(updateInterval);
+        initStatusBar();
+        initSaveLocationLabel(saveLocation);
+    }
+
+    private void initSaveLocationLabel(String saveLocation) {
+        secondaryStatusBarLabel.setText(SAVE_LOC_TEXT_PREFIX + saveLocation);
```
Comment (HansNewbie) [inline src/main/java/address/controller/StatusBarFooterController.java]: can call `updateSaveLocationDisplayHere`? So the format will always be the same.

**Comment by HansNewbie** [comment]
No more comments

**Pull request merged**